include/aikido/perception/PoseEstimatorModule.hpp

```diff
@@ -12,17 +12,18 @@
 namespace aikido {
 namespace perception {
 
-/// Instantiates the \c PerceptionModule specifically for RcnnPose.
+/// Instantiates the \c PerceptionModule specifically for PoseEstimator.
 ///
-/// It receives input from the RcnnPose running in a separate process,
-/// by subscribing to a \c visualization_msg::MarkerArray ROS topic published by
-/// the RcnnPose node. It uses a \c ObjectDatabase to resolve each marker to a
+/// It receives input from the PoseEstimator running in a separate process,
+/// by subscribing to a \c visualization_msg::MarkerArray ROS topic
+/// published by the PoseEstimator node.
+/// It uses a \c ObjectDatabase to resolve each marker to a
 /// \c dart::common::Uri, and updates the environment which is an
 /// \c aikido::planner::World.
-class RcnnPoseModule : public PerceptionModule
+class PoseEstimatorModule : public PerceptionModule
```

Comment:
What do you think about renaming this to `RosPoseEstimationModule`?

Reply:
If we want to rename it with a more ros related name, I think `RosMarkerArrayClient` would be the most clear option. If not, I personally prefer `PoseEstimatorModule`.